frontend/src/components/RecommendationList.jsx

```diff
@@ -1,51 +1,96 @@
 import React, { useState, useEffect } from "react";
 import { Link, useLocation } from "react-router-dom";
 import axios from "axios";
+import { WINE_QUERY_PARAMS, PRICE, VEGAN, FLAVOR_PROFILE } from "../constants";
 
-// comment out the 2 console logs to get a glimpse of what's in there :)
+// take out the query string from the URL and create an object out of those values
+const queryParamsToObject = (params) => {
+  const queryString = new URLSearchParams(params);
+  const queryObject = {};
+  queryString.forEach(function (value, key) {
+    queryObject[key] = value;
+  });
+  return queryObject;
+};
 
 const RecommendationList = () => {
-  const [myData, setMyData] = useState(null);
+  const [winesData, setWinesData] = useState(null);
   const navigationLocation = useLocation();
-  // console.log(navigationLocation);
+  const filters = queryParamsToObject(navigationLocation.search);
```

Comment:
Good change compared to the last commit 👍
Less state, less bugs, better code.
If you can compute a value from a state, you don't need to save the computation as state, you can always compute it again. Unless there's a huge improvement to performance in doing so.